scripts/generate_mappings.go

```diff
@@ -41,13 +42,14 @@ func main() {
 		f, err := os.Create("modules/fa-semantic-mappings/mappings.go")
 		defer f.Close()
 		if err != nil {
+			slog.Error("Error creating mappings.go", "error", err.Error())
 			panic(err)
 		}
 		f.Write([]byte(fileHeader))
-		fmt.Fprintf(f, "var Mappings = %#v\n", classMappings)
```

Comment:
this was doing Fprintf, which writes to the `f` file - this isn't a typical log & should stay